convert2rhel/logger.py

```diff
@@ -101,10 +103,13 @@ def critical(self, msg, *args, **kwargs):
 
     def debug(self, msg, *args, **kwargs):
         from convert2rhel.toolopts import tool_opts
+        from utils import format_msg_with_datetime
```

Comment:
Let's stick to the PEP8 style. 
`Imports are always put at the top of the file`
https://www.python.org/dev/peps/pep-0008/#imports